### DISCUSSION TITLE
Redirect back to items list on form "Create new item" submit action in Admin UI

### DISCUSSION THREAD

**Discussion by MurzNN** [keystonejs/keystone#6348] [Feature requests]
At now in Admin UI user after creating new item is redirected to form for edit this item. This is not logical, will be good redirect to list of items.

**Comment by gautamsi** [comment]
it should be configurable. also there is need for full page data entry which I am able to do that with custom pages

**Comment by bladey** [comment]
Thanks for the suggestion @MurzNN, it should be configurable.
